Various technical notes/transition_infos.json

```
@@ -610,7 +599,7 @@
       "exits": [
         {
           "area_id": "0x7A9B3870",
-          "area_name": "Jungle Trail (2 exits)",
+          "area_name": "Jungle Trail",
```

dark caverns and scorpions

```suggestion
          "area_name": "Jungle Trail (2 exits)",
```

---

Once again, I explicitly removed all those '(2 exits)' descriptions.
If you really, really want those back I would like to hear your reason as to why.

---

I'll resolve this conversation because there's already another one going over the same thing.